react/features/toolbox/functions.any.ts

```diff
@@ -1,14 +1,32 @@
 import { IState } from '../app/types';
+import { FEATURES_TO_BUTTONS_MAPPING } from '../base/jwt/constants';
+import { isJwtFeatureEnabled } from '../base/jwt/functions';
 
 /**
  * Indicates if the audio mute button is disabled or not.
  *
  * @param {IState} state - The state from the Redux store.
  * @returns {boolean}
  */
-export function isAudioMuteButtonDisabled(state: IState) {
+export function isAudioMuteButtonDisabled(state: IState): boolean {
```

Comment:
nit: function return type unnecessary here. Considering you return Boolean() TS can easily determine the function return type